librz/util/float/float_internal.c

```diff
@@ -0,0 +1,440 @@
+#include "rz_util.h"
+
+/**
+ * \file : Internal function for float
+ * \brief : Should be included directly in float.c
+ */
+
+static inline ut32 rz_float_info_bin32(RzFloatInfo which_info) {
+	switch (which_info) {
+	case RZ_FLOAT_INFO_BASE:
+		return 2;
+	case RZ_FLOAT_INFO_EXP_LEN:
+		return 8;
+	case RZ_FLOAT_INFO_MAN_LEN:
+		return 23;
+	case RZ_FLOAT_INFO_TOTAL_LEN:
+		return 32;
+	case RZ_FLOAT_INFO_BIAS:
+		return 127;
+	default:
+		return 0;
```

Comment:
`rz_warn_if_reached()`? Or is there a legitimate reason I miss? Same below

Reply:
ah, my mistake. `rz_warn_if_reached` should be added